SPECS/kernel/config

```diff
@@ -6517,9 +6517,9 @@ CONFIG_UDF_FS=y
 #
 # DOS/FAT/EXFAT/NT Filesystems
 #
-CONFIG_FAT_FS=m
+CONFIG_FAT_FS=y
 CONFIG_MSDOS_FS=m
-CONFIG_VFAT_FS=m
+CONFIG_VFAT_FS=y
```

Comment:
What is the difference between `m` and `y` and what does CONFIG_FAT_FS mean exactly?

Reply:
The difference between `m` and `y` is that the first makes sure the module is _available_ to be installed/uninstalled into the kernel, but the latter makes sure the module is _always_ installed into the kernel from boot up.